src/components/shared/AMMPrices/TokenAMMPriceRow.tsx

```diff
@@ -57,7 +56,7 @@ export default function TokenAMMPriceRow({
           {!exchangeSupported ? (
```

Comment:
since we're no longer using `exchangeSupported` do you think we could remove it? i'm fine with leaving it in if we expect to use it again, but it felt like more of a patch to address the concerns from folks in the community who wanted Sushi specifically.

Reply:
Yeah, good point, can remove 👍